### PR TITLE
Fixed Docstrings in geodesic Module

### LINKED ISSUE #379: Fix Docstrings in Geodesic Module
🐞 **Problem**

The docstrings in src/einsteinpy/geodesic.py is missing.
It will be great to have a good documentation of the module.

🎯 **Goal**

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by nimeshvashistha**
Fixes #379 

@shreyasbapat

### CODE REVIEW

**Comment by ritzvik** [comment]
Hi @nimeshvashistha , congrats on your first PR.

I can see you have added a file name `einsteinPY`, which is around 150,000 lines long, and maybe not required. Kindly fix it and update the PR.

Rest assured, @shreyasbapat would review it in his own time.

**Comment by nimeshvashistha** [comment]
> Hi @nimeshvashistha , congrats on your first PR.
> 
> I can see you have added a file name `einsteinPY`, which is around 150,000 lines long, and maybe not required. Kindly fix it and update the PR.
> 
> Rest assured, @shreyasbapat would review it in his own time.

Oh, I'll do the needful, thanks!

File: src/einsteinpy/geodesic.py [hunk 1]
```diff
@@ -14,7 +18,18 @@ def __init__(
...
+        step_size : float
+            Size of each increment in t
+        time : float
+            Time of start (defaults to zero seconds)
+        a : ~astropy.units.m, optional
+            Spin factor of massive body. Should be less than half of schwarzschild radius.
+        q : ~astropy.units.C, optional
+            Charge on the massive body
+        parent : Body, optional
+            The parent object of the body.
```
Comment (ritzvik) [inline src/einsteinpy/geodesic.py]: Please remove the parameters not there in `def __init__()`

Comment (ritzvik) [reply src/einsteinpy/geodesic.py]: Also add the parameter `metric`

```
metric : ~einsteinpy.metric.schwarzschild.Schwarzschild or ~einsteinpy.metric.kerr.Kerr or ~einsteinpy.metric.kerrnewman.KerrNewman
    Metric for the space-time in which geodesics are being calculated.
```

**Pull request merged**